projects/js-packages/analytics/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ### This is a list detailing changes for the Jetpack RNA Analytics package releases.
 
+## [0.1.28] - 2023-11-20
+
```

Comment:
We should update the tooling to avoid doing this. It could stick some sort of "Internal updates" entry so it's not just empty.

Too late now, of course.

Reply:
Yeah, I was wondering if that was correct, but then saw similar pre-existing entries so figured I'd leave it. But you're right.

Reply:
https://github.com/Automattic/jetpack/pull/34236